### ISSUE TITLE
`# gazelle:ignore` in a Python file does not work if it is indented

### ISSUE THREAD

**Issue by RJPercival** [bazelbuild/rules_python#1404]
# 🐞 bug report

### Affected Rule

The `# gazelle:ignore <module>` directive in a Python file.

### Is this a regression?

No, I think this bug has always been present.

### Description

If the comment is indented, then `parser.go` in the Gazelle plugin doesn't detect the directive and ignores it.

## 🔬 Minimal Reproduction

```
if True:
    # gazelle:ignore foo
    import foo
```

## 🔥 Exception or Error

```
"foo" at line 3 from "test.py" is an invalid dependency: possible solutions:
	1. Add it as a dependency in the requirements.txt file.
	2. Instruct Gazelle to resolve to a known dependency using the gazelle:resolve directive.
	3. Ignore it with a comment '# gazelle:ignore foo' in the Python file.
```

## 🌍  Your Environment

**Operating System:**

```
macOS 13.5.1
```

**Output of `bazel version`:**

```
6.3.2
```

**Rules_python version:**

```
0.25.0
```

**Comment by RJPercival** [comment]
Actually, it might not be as simple as just not recognising indented comments, as I've tried unindenting it in the case I'm dealing with, and I'm still getting the same error.
